### PR TITLE
add allowed ngnix support

### PR DESCRIPTION
**Pull request by RazcoDev**

### CODE REVIEW

File: horizon/enforcer/api.py [hunk 1]
```diff
@@ -553,6 +555,106 @@ async def is_allowed(
             )
         return result
 
+    @router.post(
```
Comment (obsd) [inline horizon/enforcer/api.py:558]: Remove

**Review by obsd — APPROVED** [review]

File: horizon/enforcer/api.py [hunk 2]
```diff
@@ -553,6 +555,51 @@ async def is_allowed(
...
+        permit_user_key: str = Header(None),
+        permit_tenant_id: str = Header(None),
+        permit_action: str = Header(None),
+        permit_resource_type: str = Header(None),
+    ):
+
+        query = AuthorizationQuery(
+            user=User(key=permit_user_key),
+            action=permit_action,
+            resource=Resource(type=permit_resource_type, tenant=permit_tenant_id),
```
Comment (omer9564) [inline horizon/enforcer/api.py:576]: don't you want to also allow having resource key ?

File: horizon/enforcer/utils/headers_utils.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+def get_case_insensitive(dictionary, key) -> str | None:
+    if isinstance(key, str):
+        return next(
+            (dictionary[k] for k in dictionary if k.lower() == key.lower()), None
+        )
+    return dictionary.get(key, None)
```
Comment (omer9564) [inline horizon/enforcer/utils/headers_utils.py:6]: remove this ?

File: horizon/enforcer/api.py (hunk 1 above)
Comment (omer9564) [reply horizon/enforcer/api.py:558]: why remove ?

File: horizon/enforcer/api.py (hunk 2 above)
Comment (RazcoDev) [reply horizon/enforcer/api.py:576]: Added

File: horizon/enforcer/utils/headers_utils.py (hunk 3 above)
Comment (RazcoDev) [reply horizon/enforcer/utils/headers_utils.py:6]: done

File: horizon/enforcer/api.py (hunk 1 above)
Comment (RazcoDev) [reply horizon/enforcer/api.py:558]: It was a redundant allowed that I duplicated.

**Pull request merged**